shell/app/modules/msp/env-overview/topology/pages/topology/utils/index.ts

```diff
@@ -74,3 +74,11 @@ export const genEdges = (data: TOPOLOGY.INode[]): Edge<TOPOLOGY.TopoEdge>[] => {
   };
   return convert(data, []);
 };
+
+export const transformCount = (num: number) => {
+  if (num >= 1000) {
+    const count = (num / 1000).toFixed(1);
+    return `${count} K`;
+  }
```

Comment:
Should use `M` if bigger than 1000K ? Could use `getFormatter` from `chart/utils`.